src/config/constants/types.ts

```diff
@@ -153,3 +154,26 @@ export type PageMeta = {
   description?: string
   image?: string
 }
+
+export enum LotteryStatus {
+  PENDING = 'pending',
+  OPEN = 'open',
+  CLOSE = 'close',
+  CLAIMABLE = 'claimable',
+}
+
+export interface LotteryTicket {
+  id: string
+  number: string
+  status?: boolean
```

Comment:
Doesn't a lottery ticket always have/need a status?

Reply:
So `status` _is_ actually always there if I want to use it... good point. But some of the components that use the interface, don't use the `status` prop (https://github.com/pancakeswap/pancake-frontend/blob/039587a9e1e87b38ab544df3880120d448d601ee/src/views/LotteryV2/components/TicketNumber.tsx#L33).

I thought it was cleaner just to make it optional rather than make a new type for components making use of only _some_ of the `LotteryTicket`'s props?